examples/website/src/shared/types.ts

```diff
@@ -334,6 +335,27 @@ export type GridBlock = BaseBlock<
   }
 >
 
+export interface FlexItemAlignment {
+  i: string
+  x: number
+  y: number
+  w: number
+  h: number
+  static?: boolean
+}
+
+export interface FlexTeaser {
+  alignment: FlexItemAlignment
+  blocks: Block[]
```

Comment:
How can a Teaser will have multiple blocks?  And is the key `blocks` or `teaser` ?

Reply:
There's a teaser in the blocks. It was a workaround to use the existing teaser render functions. Working on finding a better solution.

Reply:
@penk85 Will we be using it as blocks? It's a simple point but confusing a little, what do you think?